Makefile

```diff
@@ -31,6 +33,7 @@ croutoncursor_LIBS = -lX11 -lXfixes -lXrender
 croutonfbserver_LIBS = -lX11 -lXdamage -lXext -lXfixes -lXtst
 croutonwmtools_LIBS = -lX11
 croutonxi2event_LIBS = -lX11 -lXi
+croutonfreon.so_LIBS = -ldl
```

Comment:
I think for LIBS you can assume -ldl is needed, instead of having to specify it for each .so